processes/consumer/kafka.go

```diff
@@ -101,12 +101,19 @@ func StartConsumer(ctx context.Context, cfg config.Config, inMemDB *models.Datab
 				Brokers: cfg.Kafka.BootstrapServers(),
 			}
 
-			kafkaConsumer := kafka.NewReader(kafkaCfg)
+			kafkaConsumer := kafkalib.NewReader(kafkaCfg)
 			topicToConsumer.Add(topic, kafkaConsumer)
 			for {
 				kafkaMsg, err := kafkaConsumer.FetchMessage(ctx)
 				if err != nil {
+					if kafkalib.ShouldReload(err) {
+						if err = kafkaConsumer.Reload(); err != nil {
```

Comment:
Might want to log here that we're going to reload.

Reply:
yeah just added